packages/react-composites/tests/browser/call/HorizontalGallery.test.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { test } from './fixture';
+import { expect } from '@playwright/test';
+import { buildUrlWithMockAdapter } from './utils';
+import { dataUiId, pageClick, waitForPageFontsLoaded, waitForSelector } from '../common/utils';
+import { TestRemoteParticipant } from './TestCallingState';
+import { IDS } from '../common/constants';
+
+const defaultTestRemoteParticipants: TestRemoteParticipant[] = [
+  {
+    displayName: 'Paul Bridges',
+    isSpeaking: true,
+    isVideoStreamAvailable: true
+  },
+  {
+    displayName: 'Eryka Klein'
+  },
+  {
+    displayName: 'Fiona Harper',
+    isVideoStreamAvailable: true
+  }
+];
+
+test.describe('HorizontalGallery tests', async () => {
+  test('HorizontalGallery should have 1 audio participant', async ({ pages, serverUrl }) => {
+    const page = pages[0];
+    await page.goto(
+      buildUrlWithMockAdapter(serverUrl, {
+        remoteParticipants: defaultTestRemoteParticipants
+      })
+    );
+    // Click off the screen to turn away initial aria label
```

Comment:
Don't we already have a helper function for this? If not, let's create on `clickOffPage()` so that the function name serves instead of repeating this comment everywhere.